tests/apps/courses/test_templates_course_detail_opengraph.py

```diff
@@ -185,3 +187,35 @@ def test_open_graph_course_description_no_introduction(self):
         response = self.client.get(url)
 
         self.assertNotContains(response, "og:description")
+
+    # pylint: disable=no-self-use
+    def mock_storage_url(self, name):
```

Comment:
I'll move this function above test methods in order to split utils and tests.

Reply:
I think that the mock is too specific for the test case, so I refactor the function to be lambda inline. 

```python
@mock.patch.object(
        FileSystemStorage,
        "url",
        lambda storage_instance, name: f"https://test-bucket.s3.amazonaws.com/{name:s}",
    )
    def test_open_graph_course_meta_og_image_media_storage_s3(self):
```

@jbpenrath  are you ok with this solution ?